.kokoro/samples-test.sh

```diff
@@ -31,27 +31,12 @@ if [ -f .kokoro/pre-samples-test.sh ]; then
     set -x
 fi
 
-if [ -f samples/package.json ]; then
-    npm install
+npm install
 
-    # Install and link samples
-    cd samples/
-    npm link ../
-    npm install
-    cd ..
+# Install and link samples
+cd samples/
+npm link ../
+npm install
+cd ..
 
-    npm run samples-test
-fi
-
-# codecov combines coverage across integration and unit tests. Include
-# the logic below for any environment you wish to collect coverage for:
-COVERAGE_NODE=10
```

Comment:
this is odd, it seems to revert a bunch of configuration, which is landed upstream; I wonder if perhaps you're running an older version of `synthtool` -- mind running a local version checked out from master?

Reply:
Re-running from Master removed all these changes.